framework/src/main/java/org/tron/core/services/http/Util.java

```diff
@@ -535,7 +534,9 @@ private static String checkGetParam(HttpServletRequest request, String key) thro
       if (StringUtils.isBlank(contentType)) {
         return null;
       }
-      if (APPLICATION_JSON.toLowerCase().contains(contentType)) {
```

Comment:
Should the contentType to lower case too ?

Reply:
defaults to lowercase.